client/src/legacy/CKANRegistryDisplay-entwine.js

```diff
@@ -0,0 +1,11 @@
+import jQuery from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import CKANRegistryDisplay from '../components/CKANRegistryDisplay';
+
+(($) => $(() => {
```

Comment:
The filename indicates that this is entwine code, when in fact it's not using entwine. This particular line is also a bit confusing to read - we usually do this:

```js
jQuery.entwine('ss', ($) => {
```

You could do the same thing with or without the entwine bit =)

Reply:
This is currently without entwine, you're right. Got carried away copying the convention from the other files in the folder 😅